### DISCUSSION TITLE
Using "AND" and "OR"-like operators in issue searches

### DISCUSSION THREAD

**Discussion by henrik-jensen** [community/community#23976] [Projects and Issues]
<p>I want to narrow down my search hits on issues but it seems hard to restrict without these operators ("AND, “OR”) in some format. How do one accomplish that on github?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>If you have an example, we may be able to help you fine tune it.</p>
<p>You should be able to use those operators - for instance:</p>
<p>(<a href="https://github.com/github/fetch/issues">example repo here</a>)</p>
<p><code>is:issue is:open function OR content</code></p>
<p>6 issues match</p>
<p><code>is:issue is:open function content</code></p>
<p>1 issue (default is AND)</p>
<p><code>is:issue is:open function</code></p>
<p>4 issues</p>
<p><code>is:issue is:open content</code></p>
<p>3 issues</p>

**Comment by henrik-jensen** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a> : Thanks for the welcome  and answer. :slight_smile:</p>
<p>Here are some examples:<br>
(My goal was to see if I could find the reason the vscode built-in extension ‘emmet’ was disabled for markdown files, before enabling markdown for emmet my self).</p>
<p>On vscode issues Query <a href="https://github.com/microsoft/vscode/issues?q=emmet+AND+markdown+" rel="noopener nofollow ugc">emmet AND markdown</a><br>
results in → 7 open  213 closed.</p>
<p>But examining some of the actual links some has only one of the search words:</p>
<ul>
<li>Example: issue <a href="https://github.com/microsoft/vscode/issues/119120" rel="noopener nofollow ugc">#119120 - Snippets of Emmet stopped working</a> only “emmet” is found not “markdown” so this search is more like an <strong>OR</strong> search</li>
</ul>
<p>and some has neither, (sorry I’ve stumbled into the 2 links restriktion pr. post for new members)</p>
<ul>
<li>Example: <em>Issue <span class="hashtag">#118796</span> - JavaScript function not working</em>, has <strong>neither search words</strong> but has ‘<em>and</em>’ hits.<br>
Using space instead of <strong>AND</strong>: “emmet markdown” still commes up with 7 open . 213 closed and issue <em><span class="hashtag">#118796</span> - JavaScript function not working</em> (having no actual hits), are still listed as a result.</li>
</ul>

**Comment by canuckjacq** [comment]
<aside class="quote no-group" data-username="henrik-jensen" data-post="3" data-topic="168829">
<div class="title">
<div class="quote-controls"></div>
 henrik-jensen:</div>
<blockquote>
<p>Example: issue <a href="https://github.com/microsoft/vscode/issues/119120">#119120 - Snippets of Emmet stopped working</a> only “emmet” is found not “markdown” so this search is more like an <strong>OR</strong> search</p>
</blockquote>
</aside>
<p><code>markdown</code> is in that issue - but it’s under the summary tags, so you’d have to expand those all before a browser search function would locate it. It’s in the original post, and in a comment.</p>
<p>Expand this:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/c/8cfe4dc10b6e8945b6c4baf840254da165e6f959.png" data-download-href="/uploads/short-url/k7hGIQecHbqepzcv6Hju3CmsRsZ.png?dl=1" title="Screenshot 2021-03-18 at 16.27.13"><img src="https://user-images.githubusercontent.com/11161530/181080478-bde76717-e5a9-406e-b68f-0cbf8f6f6072.png" alt="Screenshot 2021-03-18 at 16.27.13" data-base62-sha1="k7hGIQecHbqepzcv6Hju3CmsRsZ" width="345" height="83" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8cfe4dc10b6e8945b6c4baf840254da165e6f959_2_345x83.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8cfe4dc10b6e8945b6c4baf840254da165e6f959_2_517x124.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8cfe4dc10b6e8945b6c4baf840254da165e6f959_2_690x166.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8cfe4dc10b6e8945b6c4baf840254da165e6f959_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-03-18 at 16.27.13</span><span class="informations">734×178 5.29 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>To see this:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/f/0f37ebf6df1b6ec1cd2c825affea16fc8d1f508b.png" data-download-href="/uploads/short-url/2aCYnngO2aJ2DIBHWmZIQn8u8UH.png?dl=1" title="Screenshot 2021-03-18 at 16.27.04"><img src="https://user-images.githubusercontent.com/11161530/181080479-2d153959-2f9b-482e-8b9a-9cdd9982ec75.png" alt="Screenshot 2021-03-18 at 16.27.04" data-base62-sha1="2aCYnngO2aJ2DIBHWmZIQn8u8UH" width="345" height="177" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f37ebf6df1b6ec1cd2c825affea16fc8d1f508b_2_345x177.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f37ebf6df1b6ec1cd2c825affea16fc8d1f508b_2_517x265.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f37ebf6df1b6ec1cd2c825affea16fc8d1f508b_2_690x354.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f37ebf6df1b6ec1cd2c825affea16fc8d1f508b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-03-18 at 16.27.04</span><span class="informations">1060×546 45.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by henrik-jensen** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a>:<br>
Ah haaaa - solved :+1:  Thanks.<br>
There are 2 ice creams to choose from, so pick you favorite or have them all,- you deserve it :icecream: :ice_cream: :smile:</p>
<p>Edit:<br>
Changed the solution to <a class="mention" href="/canuckjacq">@canuckjacq</a>’s first post for discoverability as that has the answer to my question while the other pointed out my false assumption on the search results, and a shallow reading of that as a solution might not make much sense with regard to the question.</p>
